binders/rabbit-binder/spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/stream2/RabbitStreamBinderModuleIntegrationTests.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binder.rabbit.stream2;
```

Comment:
Why so strange name for the package?

Reply:
Because if I put it in the same package `stream`; the other `@Configuration` class gets loaded too, causing bean override and non-unique bean problems.

Reply:
I see. Probably that's why Spring Boot is not recommend to use `@SpringBootApplication` in the unit tests.
Rather `@EnableAutoConfiguration`.
In other words: we need to exclude a `@ComponentScan` somehow which is there by default for the current package on the `@SpringBootApplication`.

But yeah... That's minor and you can disregard my call.